### ISSUE TITLE
Rotate the tilde glyph to the usual position

### ISSUE THREAD

**Issue by dreamer** [microsoft/cascadia-code#216]
Before I'll proceed with my feature request, let me say - thank you for this font, it is very legible and overall pleasant to use, especially after you included PowerLine characters :)

# Description

Font designer made a stylistic choice regarding tilde (`~`) glyph in Cascadia:

- Glyph is vertically centered (this is ok, great for a font designed for coding)
- Tilde is "rotated", to make glyph terminals parallel to the mean line (this is not ok, you wouldn't rotate any other glyph just to make it "more pleasant" to look at)

This is somewhat annoying and unexpected, especially since I see this glyph every single time I open my terminal emulator…

In other words, this is how it looks like right now:

![cascadia-rotated-tilde](https://user-images.githubusercontent.com/3967/70865895-d3ddb580-1f62-11ea-937c-13fc45fce06e.png)

At larger size it does not look so bad, but at smaller size, the middle part of a tilde glyph ("bar") looks horizontal, making it look like some different character altogether.

And I would prefer if it looked somewhat like this:

![tilde-good](https://user-images.githubusercontent.com/3967/70865912-e8ba4900-1f62-11ea-8497-d351c4e64e61.png)

Excuse my poor attempt here, I merely "rotated" back part of raster image. The point is to make the tilde look "balanced".

**Comment by aaronbell** [comment]
Thank you for the feedback! I can totally understand your point of view that the tilde character may seem a bit unusual compared to how one might expect. There's two points I'd like to raise. 

1) In talking about the ascii tilde character (~), we have to also consider the diacritic tilde character (such as on ñ). Now, diacritics can vary quite a bit in design and character depending on the design of the font, and in Cascadia Code, the ~ definitely falls on the 'exuberant' end of the scale.  However, this is not a situation where I've randomly chosen to rotate the design. 

Here's an example of a typeface designed by Geronimo Gil in 1780 (fun fact, it was created for an edition of _Don Quixote_ in 1780!). Note the playful upward angle of the tilde in the word 'añadió'.
![grakqqqm94uh](https://user-images.githubusercontent.com/8460297/70872785-32089980-1f5f-11ea-81f4-9a8231d9bb24.png)

Here's another example—a design called Merlo, from 1799. Note in the word 'año' in the second line of the left example, which similarly has an upward motion. 
![Merlo_Originals](https://user-images.githubusercontent.com/8460297/70872716-ee159480-1f5e-11ea-9978-05c8fa84b04d.jpg)

The tilde in Cascadia Code doesn't quite go to the extremes of the examples shown above, but I wanted to show how this character can take on a variety of forms. 

2) When considering how to ensure that a shape renders well on screen at low resolution, I wanted to be careful to emphasize the instroke (left-side) and outstroke (right side) of the tilde shape. My fear with the tilde character is that under low resolutions it can either (a) become flattened into a macron / dash like shape, or (b) becomes totally blurry and without shape (thusly unrecognizable). To resolve this, I wanted to emphasize the verticals, and have the central bar flatter (as it will render more sharply / crisply on a pixel grid). That is why it has the shape that it does. I also like that it gives the tilde a bit of flair and fun that's a little unexpected, which is part of the design concept of Cascadia Code.

The above aside, as I reviewed the various tilde shapes throughout the font, I note that there's a bit of an inconsistency in the design shape. Some are a bit more traditionally curvy, others more flat. I should go through and iron these out—though I should say I'm leaning toward the flatter form ;). 

In any case, I'm interested to know if other folks have a similar perspective on the tilde (as I consider how to move forward with it), so I'll leave this open for now. Thanks again!

**Comment by mdtauk** [comment]
It is a question of character, as well as legibility.  A flatter Tilde will render clearer as the horizontal stroke will sit on a pixel grid later.

But flatter or more angled, its good that this is done consistently not just through other forms of Tilde like the diacritic, but also in ligatures and other glyphs which have similar forms and curvatures.

**Comment by rdnlsmith** [comment]
I personally would also prefer the more typical orientation. I'm getting used to it, but at my usual font size I do think it looks a little unbalanced, especially when juxtaposed with a unix-style directory separator:

![image](https://user-images.githubusercontent.com/4039042/71178585-82704780-223c-11ea-85c4-9444d7f3e7e8.png)

(I think the Windows-style backslash kind of helps to counter-balance the tilde, so it doesn't look as strange to me in that context.)

@aaronbell I do want to say that I can appreciate the reasons you've stated for the tilde's current design, and also that I respect your vision as the font designer. Cascadia Code very quickly became my favorite font for programming!

**Comment by fitojb** [comment]
If people keep asking to tone down what makes this font special, you might as well redraw Consolas and be done with it.
